src/utils.js

```diff
@@ -40,7 +47,7 @@ const psTreePromised = util.promisify(psTree);
  * Creates an instance of ApifyClient using options as defined in the environment variables.
  * This function is exported to enable unit testing.
  *
- * @returns {*}
+ * @returns {ApifyClient}
```

Comment:
since `ApifyClient` is untyped, it's erroring when compiling 
![image](https://user-images.githubusercontent.com/461084/73303921-0c291280-41f6-11ea-8dcf-d22741b834c9.png)
so it's best to set to an `{Object}` until it becomes typed (I have a preliminary typings for it, but the API surface is really big)

Reply:
Done.